### DISCUSSION TITLE
Downtime for https://opnxng.com

### DISCUSSION THREAD

**Discussion by extremelyonline** [searxng/searx-instances#356] [Report instance downtime]
My instance https://opnxng.com has went down since yesterday. I suspect this is related to a recent Debian update. I will troubleshoot and resolve the issue as soon as possible. Sorry for the downtime.

**Comment by unixfox** [comment]
Thank you for the notification

**Comment by extremelyonline** [comment]
Seems to be fine after two weeks. It was a problem related to another Docker container.
